config/test.json

```diff
@@ -104,6 +104,7 @@
 		"reader/full-errors": true,
 		"reader/list-management": true,
 		"reader/public-tag-pages": true,
+		"readymade-templates/showcase": true,
```

Comment:
I would set it to false on test envs for now. Just in case we interfere with anything and block deployments. We will eventually need to enable it before we release though so feel free to leave it as it is.